packages/block-library/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 2.1.8 (unreleased)
```

Comment:
If it's a new feature, it should be `2.2.0`

Reply:
Yeah, I wasn't sure as it is a small non breaking change, more of a small enhancement. Will change this tonight.